sweetest/src/main/java/com/mysugr/sweetest/framework/base/BaseSteps.kt

```diff
@@ -6,7 +6,7 @@ import com.mysugr.sweetest.framework.context.TestContext
 import kotlinx.coroutines.CoroutineScope
 import kotlin.coroutines.CoroutineContext
 
-interface Steps : CoroutineScope
```

Comment:
I know your reasons for removing this, however, it could be a bit weird that in a test one can write:
```
class BlaTest {
  @Test
  fun `...`() = runBlockingTest {
    launch {
      steps.doSomething()
    }
  }
}
```
...`launch` without explicitly using `coroutineScope`.

And in the steps class:

```
class BlaSteps {
  fun doSomething() {
    coroutineScope.launch { 
      // do something more
    }
  }
}
```
... here the `coroutineScope` is required.


It's a bit inconsistent. Just food for thought.

Reply:
hm, possible solutions:

- just offer `coroutineScope` for steps classes
- don't offer it at all and in steps class have an extension `CoroutineScope.doSomething()` that receives the coroutine scope from the test class / `testCoroutine`.

nonetheless, i consider the extension a convenience thing. question is whether we really need to do something about it and it would have a positive effect or just be annoying...